### PR TITLE
Fix: Flex overflow issues in Safari/IE

### PR DESCRIPTION
**Pull request by omgaz**
Broken flex containers in IE and Safari

screen shot 2016-04-08 at 11 55 21

Fix as per documentation:
As per http://philipwalton.com/articles/normalizing-cross-browser-flexbox-bugs/

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM but its not passing linting

File: src/styles/alexandria/GridCell.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 $cell-padding: 10px;
 
 .grid-component-cell {
-  flex-grow: 1;
```
Comment (omgaz) [inline src/styles/alexandria/GridCell.scss]: check this? do we need this on the -stretch?

**Comment by omgaz** [comment]
Tested with Ryan in Firefox/Chrome/Safari/IE.

**Comment by etaoins** [comment]
:rocket:

**Pull request merged**